databuilder/sqlalchemy_utils.py

```diff
@@ -150,8 +151,8 @@ class TemporaryTable(Table):
     with the queries that use them and they no longer need to be manged out-of-band.
     """
 
-    setup_queries: list[Executable]
-    cleanup_queries: list[Executable]
+    setup_queries: Sequence[Executable]
+    cleanup_queries: Sequence[Executable]
```

Comment:
We need these to be covariant because we put subclasses of `Executable` into them.